memgpt/config.py

```diff
@@ -178,6 +178,7 @@ def save(self):
         # CLI defaults
         config.add_section("defaults")
         config.set("defaults", "model", self.model)
+        config.set("defaults", "context_window", str(self.context_window))
```

Comment:
@sarahwooders I casted to a `str` here because I was getting this error:

```
File "/Users/loaner/dev/MemGPT-public/MemGPT/memgpt/cli/cli_config.py", line 217, in configure
    config.save()
  File "/Users/loaner/dev/MemGPT-public/MemGPT/memgpt/config.py", line 181, in save
    config.set("defaults", "context_window", self.context_window)
  File "/opt/homebrew/Cellar/python@3.11/3.11.6_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/configparser.py", line 1219, in set
    self._validate_value_types(option=option, value=value)
  File "/opt/homebrew/Cellar/python@3.11/3.11.6_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/configparser.py", line 1204, in _validate_value_types
    raise TypeError("option values must be strings")
TypeError: option values must be strings
```

Reply:
ah yeah I saw that too -- will be fixed here https://github.com/cpacker/MemGPT/pull/422/files#diff-abeb817d2a7dcff0c6189de70c935ff700db96e8e7e2f33f984e5ab0cc188dd1R111 but lets merge this for now